wallet/src/types.rs

```diff
@@ -151,14 +153,16 @@ pub struct VttParams {
     pub value: u64,
     pub fee: u64,
     pub time_lock: u64,
+    pub weighted_fee: Option<u64>
```

Comment:
This is totally the place for explaining that this is nanowits per weight unit. Also could use some clarification on why this is optional.

Reply:
We made it optional not to break the API. But we can make this non-optional if we believe it is better

Reply:
So essentially this PR assumes that if weighted_fee is None, 'fee' is utilized as the tx fee without taking into account the transaction weight. If weighted_fee is Some, then weighted_fee is multiplied by the transaction weight

Reply:
I like the idea of throwing an error if both fee fields are present. Maybe we could do something like one field to specify the type of fee, and another one with the value. :)

Reply:
implemented